Cargo.toml

```diff
@@ -30,13 +29,23 @@ num-derive = "0.2"
 bitflags = "1.0.4"
 serial = { version = "0.4", optional = true }
 serde = { version = "1.0.101", optional = true }
-
-[dependencies.byteorder]
-version = "0.5.3"
-optional = true
-
+byteorder = { version = "0.5", optional = true }
 
 [features]
+"ardupilotmega" = []
+"asluav" = []
+"autoquad" = []
+"matrixpilot" = []
+"minimal" = []
+"paparazzi" = []
+"python_array_test" = []
+"slugs" = []
+"standard" = []
+"test" = []
+"ualberta" = []
+"uavionix" = []
+"icarous" = []
+
```

Comment:
A final note, will common prefix be everything ? What happens if one is selected over the others or if someone want to change the dialect ?

Reply:
I don't really understand your question. The `common::` prefix only applies to messages and data types that are defined in the common message set.

Reply:
The reason that I consider this a breaking change is because some user-facing types, such as `MavConnection<T>` are now generic, where they weren't before.

Reply:
I see it now, thanks for the explanation!